index.html

```diff
@@ -277,6 +268,28 @@ <h4 class="media-text">LET’S CONNECT</h4>
             </div>
          </div>
         </section>
+        <section class="contact-section">
+          <div class="contact-container">
+            
+            <div class="c-title-container">
+              <h2 class="c-title">Contact me</h2>
+            </div>
+            <div class="c-desc-container">
+              <p class="c-desc">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            </div>
+            <div class="form-container">
+              <form class="contact-form" action="https://formspree.io/f/xdojgvqp" method="post">
+                <input type="text" name="Full Name" placeholder="Enter Full-name" class="customer-info" minlength="2" maxlength="30" required >
+                <input type="email" name="Email" placeholder="abc@example.com" class="customer-info" required>
+                <textarea name="Message" rows="6" placeholder="Enter your message here" class="customer-message" required minlength="15" maxlength="300"></textarea>
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you increase your maxlength  value from 300 to 500 because of the project requirements.